lib/exception/CmdException.ts

```diff
@@ -0,0 +1,11 @@
+import {IProcess} from '../interfaces';
+
+export class CmdException extends Error {
```

Comment:
Instead of CmdException lets use `ReportException` that has following fields:
- message: string
- type: string
- payload?: any